### PR TITLE
updated calling rust from frontend example

### PR DESCRIPTION
**Pull request by rohanshetty-dev**
imported function and variable declared had the same name. So, I changed the variable name

### CODE REVIEW

**Comment by FabianLars** [comment]
The duplicate function name is intention, because they _are_ the same functions.

It's supposed to show you how to do things with either the npm package or the global tauri object.

File: docs/guides/features/command.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Now, you can invoke the command from your JS code:
 // With the Tauri API npm package:
```
Comment (lorenzolewis) [inline docs/guides/features/command.md]:
```suggestion
// When using the Tauri API npm package:
```

File: docs/guides/features/command.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Now, you can invoke the command from your JS code:
...
 import { invoke } from '@tauri-apps/api/tauri'
 // With the Tauri global script, enabled when `tauri.conf.json > build > withGlobalTauri` is set to true:
```
Comment (lorenzolewis) [inline docs/guides/features/command.md]:
```suggestion
// When using the Tauri global script (if not using the npm package
// Be sure to set `build.withGlobalTauri` in `tauri.conf.json` to true
```

File: docs/guides/features/command.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ Now, you can invoke the command from your JS code:
...
-const invoke = window.__TAURI__.invoke
+const invokeTauri = window.__TAURI__.invoke
```
Comment (lorenzolewis) [inline docs/guides/features/command.md]:
```suggestion
const invoke = window.__TAURI__.invoke
```

**Comment by lorenzolewis** [comment]
@FabianLars  how about we update the phrasing on the comments (committed it on here). That may help to clear up some confusion

**Comment by FabianLars** [comment]
yeah that works, long overdue tbh

**Review by FabianLars — APPROVED** [review]

**Pull request merged**